apps/updatenotification/lib/BackgroundJob/ResetToken.php

```diff
@@ -26,24 +27,31 @@ public function __construct(
 		ITimeFactory $time,
 		private IConfig $config,
 		private IAppConfig $appConfig,
+		private LoggerInterface $logger,
 	) {
 		parent::__construct($time);
-		// Run all 10 minutes
-		parent::setInterval(60 * 10);
+		// Run once an hour
+		parent::setInterval(60 * 60);
 	}
 
 	/**
 	 * @param $argument
 	 */
 	protected function run($argument) {
 		if ($this->config->getSystemValueBool('config_is_read_only')) {
+			$this->logger->debug('Skipping `updater.secret` reset since config_is_read_only is set', ['app' => 'updatenotification']);
 			return;
 		}
 
 		$secretCreated = $this->appConfig->getValueInt('core', 'updater.secret.created', $this->time->getTime());
```

Comment:
```suggestion
		$secretCreated = $this->appConfig->getValueInt('core', 'updater.secret.created');
		if ($secretCreated === 0) {
			...
```

Should handle the "does not even exist" case.
Or should we even always try the delete of the `updater.secret`, when the timestamp is missing?